src/material/tabs/tab-group.ts

```diff
@@ -201,22 +204,28 @@ export class MatTabGroup implements AfterContentInit, AfterContentChecked, OnDes
   @Input({transform: booleanAttribute})
   preserveContent: boolean = false;
 
-  /** Background color of the tab group. */
+  /**
+   * Background color of the tab group.
+   * @deprecated The background color should be customized through Sass theming APIs.
+   * @breaking-change 20.0.0 Remove this input
+   */
   @Input()
   get backgroundColor(): ThemePalette {
     return this._backgroundColor;
   }
 
   set backgroundColor(value: ThemePalette) {
-    const classList: DOMTokenList = this._elementRef.nativeElement.classList;
+    if (ENABLE_BACKGROUND_INPUT) {
```

Comment:
nit: maybe we should log a warning here if it's disabled so that internal users don't wonder why it does nothing?

Reply:
Totally - in fact I'd take this one step further and throw an Error to say it's not a valid input. I wish it was namespaced more specifically just in case it's used by a sibling directive, but I don't expect that to be likely.

Reply:
The Error will help perform a migration to catch all usages before we disable it